### DISCUSSION TITLE
Superset Dashboard Cross Filtering on Selected Columns

### DISCUSSION THREAD

**Discussion by pardeepnaik** [apache/superset#23412] [Q&A / Help]
I enabled cross-filtering for a chart in Superset. As expected it's filtering other charts data when I am clicking on any column value, however, I am expecting that only one column should be clickable, which will filter other charts data. E.g. My chart (table) have 3 columns (col 1, col2 and col3), currently if I am clicking on any value of col1, col2 or col3, its filtering other charts accordingly. However, I want that other charts should only be filtered when I click on any value of col1 only (col2 & col3 shouldn't affect other charts when clicked, or better they shouldn't be clickable). Is it possible in Superset?

Thanks in advance!

**Comment by fmannhardt** [comment]
I have the same questions. Often there is a kind of index column on which it makes sense to filter, other columns show data values for which cross filtering makes little sense.

**Reply by fmannhardt** [reply]
Just as an idea how this could be done without extra configuration. In the datasets it is possible to mark a colum as "Is filterable", I would have expected that if this property to FALSE, then it should also not be used in cross filtering as it is correctly not showing up in the chart filtering. Maybe this makes this feature much easier to implement.
